### ISSUE TITLE
Wrong devID ends up being set

### ISSUE THREAD

**Issue by Ghawken** [FlyingDiver/Indigo-GeoFence#1]
https://github.com/FlyingDiver/Indigo-GeoFence/blob/cac20c321052c6fa5c517954878605f24912ec92/GeoFence.indigoPlugin/Contents/Server%20plugin/plugin.py#L179

the devID in self.deviceList - continues advancing and ends up being incorrect if devices exists and more than one device.
A break fixes, although may be more elegant way to resolve.

```
        if self.deviceList:
            for devID in self.deviceList:
                if self.deviceList[devID]['address'] == deviceAddress:
                    self.logger.debug(f"Found userLocation device: {self.deviceList[devID]['name']}")
                    foundDevice = True
                    {add break here}
        if not foundDevice:
            self.logger.warning(f"Received {event} from {deviceAddress} but no corresponding device exists")
            if self.createDevice:
                devID = self.deviceCreate(sender, location)
            else:
                devID = None
        if not devID:
            return
```

**Comment by FlyingDiver** [comment]
Yeah, that doesn't look right.  I don't think I wrote this code originally.  I'll fix.

**Comment by FlyingDiver** [comment]
Fixed in 2022.1.2, I hope.

**Comment by FlyingDiver** [comment]
Try:  https://github.com/FlyingDiver/Indigo-GeoFence/releases/tag/2022.1.2
